apps/api/src/app/app.controller.ts

```diff
@@ -101,6 +105,18 @@ export class AppController {
         hashid: hashid,
       })
       .subscribe();
+
+      this.telemetryService.sendEvent(
+        this.configService.get<string>("POSTHOG_DISTINCT_KEY"),
+        `Redirected Link`,
+        {
+          linkId:redirectedLink.id,
+          routeName: `/${hashid}}`,
+          queryParams : redirectedLink?.params,
```

Comment:
this isn't what I meant by `params`. I was referring to the query params that are coming from the redirect url.

Reply:
example: 
redirectUrl: `localhost:3000?abc=123&def=456`
```
queryParams: {
   abc: 123,
   def: 456
}
```

Reply:
good point. for now lets just remove queryparam field from here. we will see later what needs to be done.

Reply:
@yuvrajsab i have remove it